packages/example-getting-started/docs/7-controller.md

```diff
@@ -19,6 +19,78 @@ In addition to creating the handler functions themselves, we'll also be adding
 decorators that setup the routing as well as the expected parameters of
 incoming requests.
 
+First, we need to define our basic controller class as well as plug in our
+repository, which we'll use to perform our operations against the datasource.
+
+#### src/controllers/todo.controller.ts
+```ts
+import {repository} from '@loopback/repository';
+import {TodoRepository} from '../repositories/index';
+
+export class TodoController {
+  constructor(
+    @repository(TodoRepository.name) protected todoRepo: TodoRepository,
+  ) {}
+}
+```
+
+The `@repository` decorator will retrieve and inject an instance of the
+`TodoRepository` whenever an inbound request is being handled. The lifecycle
+of your controller object is typically a per-request lifecycle, which means that
+a new controller instance is created for each request. As a result, we want to
+inject our `TodoRepository` since the creation of these instances is more
+complex and expensive than making new controller instances.
+
+>**NOTE**: You can customize the lifecycle of *all* bindings in LoopBack 4!
+>Controllers can easily be made to use singleton lifecycles to minimize startup
+>costs. For more information, see the
+>[Dependency injection](http://loopback.io/doc/en/lb4/Dependency-injection.html)
+>section of our docs.
+
+Now that we have the repository wireup, let's create our first handler function.
+
+#### src/controllers/todo.controller.ts
+```ts
+import {post, param} from '@loopback/openapi-v2';
+import {HttpErrors} from '@loopback/rest';
+import {TodoSchema, Todo} from '../models';
+import {repository} from '@loopback/repository';
+import {TodoRepository} from '../repositories/index';
+
+export class TodoController {
+  constructor(
+    @repository(TodoRepository.name) protected todoRepo: TodoRepository,
+  ) {}
```

Comment:
A line break here would be great, but that change probably also needs to be made in the actual file itself, so idk.

Reply:
Our lint rules explicitly forbid it via Prettier (it'll collapse it again on autofix)